types/methods/list.d.ts

```diff
@@ -0,0 +1,33 @@
+import type {Execa} from './main-async.js';
+import type {ExecaNode} from './node.js';
+import type {ExecaScriptCommon, ExecaScriptSync} from './script.js';
+
+// The types seem to fail unless type aliases are used.
+// We do not know why.
```

Comment:
I cannot explain this. 
The types fail unless type aliases are used. :thinking: